tests/rating/chai-jquery.js

```diff
@@ -0,0 +1,231 @@
+(function (chaiJquery) {
```

Comment:
I am afraid that we e can't accept the additional testing framework dependency in our suite; we already have qUnit in place.